<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopEntriesWindow</class>
 <widget class="QWidget" name="ShopEntriesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Order</string>
  </property>
  <property name="windowIcon">
   <iconset theme="x-office-calendar"/>
  </property>
  <widget class="QTableView" name="tableView_orderSelectMovie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_orderSelectCustomer">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_paymentDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>741</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="title">
    <string>Payment details</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_pMethod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pAmount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pDeposit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Method:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Deposit:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_pIsCompleted">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Completed</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pConfirm">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm payment</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_orderSelectPayment">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a movie to order:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a customer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of days:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_finishOrder">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>500</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Finish order</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Days">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
